Algebraic_kernel_d/examples/Algebraic_kernel_d/CMakeLists.txt

```diff
@@ -18,20 +18,17 @@ endif()
 include( ${CGAL_USE_FILE} )
 
 
-find_package( Boost REQUIRED )
+# Boost and its components
+find_package( Boost QUIET )
 if ( NOT Boost_FOUND )
   message(STATUS "This project requires the Boost library, and will not be compiled.")
-  return()  
+  return()
 endif()
 
-
-find_package( MPFI REQUIRED )
-if ( NOT MPFI_FOUND )
-  message(STATUS "This project requires the MPFI library, and will not be compiled.")
-  return()  
+find_package( MPFI QUIET )
```

Comment:
@lrineau Is it expected that CMake behaves differently when I leave out line 28 (separate find_package call for MPFI) and add MPFI in line 10? In that case, at least on my setup, the MPFI_USE_FILE is not loaded AFAICS.